nbconvert/exporters/webpdf.py

```diff
@@ -38,6 +40,11 @@ def _check_launch_reqs(self):
     def run_pyppeteer(self, html):
         """Run pyppeteer."""
 
+        temp_file = tempfile.NamedTemporaryFile(suffix=".html", delete=False)
```

Comment:
We should probably rely on the contextmanager somehow to cleanup the temporary file. I *think* right now, it's possible for a crash to leave this HTML file around

Reply:
Yeah that is a possibility, but temp files are stored in temporary storage, so dangling files should not be an issue long term. I did it this way because it is important to close the file to ensure the whole data is transfered to the browser process without any buffering issues.

Reply:
True! But you can call `temp_file.fluash()` to ensure that.

Reply:
Not really. According to the documentation if the file is not closed, there is no guarantee that the file can be opened and read by another process in some operating systems (NT derived I think)